Readme.md

```diff
@@ -59,37 +59,55 @@ This is very important; not doing it either leads to compiler errors, or even wo
 struct nk_context ctx;
```

Comment:
Not sure why we are updating the example in the readme. IMO readme examples should be very small and give the reader a taste of how nuklear is to use. Not sure this update really makes it any better

Reply:
It shows elements of nuklear that are often going to be used in applications, but I guess it's preference

Reply:
I usually tend to prefer readme examples to be **very** small. In our case we're pretty much covered by everything in `example/` directory. So here I won't advocate for extending the current example.

I'm though fine with extending if it's just about 1-5 SLOC which add and I quote "often used elements". That's not the current case though.